src/react-apps/applications/ux-editor/src/reducers/workflowReducer/index.ts

```diff
@@ -0,0 +1,8 @@
+import { WorkflowSteps } from '../../containers/WorkflowStep';
```

Comment:
Should `WorkflowSteps` be defined in global.ts instead?

Reply:
https://github.com/Microsoft/TypeScript/issues/14975

Just seemed hacky to make it availibale in global.ts. Seems cleaner to keep it in Workflow for now.